configure.ac

```diff
@@ -56,6 +56,27 @@ AC_PROG_RANLIB
 AC_PROG_MKDIR_P
 AC_PROG_CXX
 
+# Python configuration
```

Comment:
There are indeed quite a few changes to the configure/autostuff. Anything of consequence that folks should be aware of, such as new configure options, paths, etc.?